genesyscloud/resource_genesyscloud_widget_deployment_test.go

```diff
@@ -0,0 +1,215 @@
+package genesyscloud
+
+import (
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mypurecloud/platform-client-sdk-go/v56/platformclientv2"
+	"testing"
+)
+
+type widgetDeploymentConfig struct {
+	resourceID             string
+	name                   string
+	description            string
+	authenticationRequired string
+	disabled               string
+	flowID                 string
+	allowedDomains         []string
+	clientType             string
+	webChatSkin            string
+	authenticationUrl      string
+}
+
+func generateWidgetDeployV2(widgetDeploymentConfig *widgetDeploymentConfig) string {
+	return fmt.Sprintf(`resource "genesyscloud_widget_deployment" "%s" {
+		name = "%s"
+		description = "%s"
+		flow_id = "%s"
+		client_type = "%s"
+		authentication_required = %s
+        disabled = %s
```

Comment:
Some minor formatting issues here with strings

Reply:
Should be fixed.   Didnt show up as misformatted in Goland, but replaced spaces with tabs